apis/ScreenCapture/IScreenCapture.h

```diff
@@ -58,6 +58,15 @@ namespace WPEFramework
             // @returns Core::hresult
             virtual Core::hresult UploadScreenCapture(const string& url , const string& callGUID , Result &result /* @out  */ ) = 0;
             /**********************uploadScreenCapture() - end*********************************/
+
+            /**********************sendScreenshot() - start*******************************/
+            // @text sendScreenshot
+            // @brief Takes a screenshot and uploads it to the specified URL
+            // @param callGUID -  A unique identifier of a call. The identifier is used to find a corresponding uploadComplete event
+            // @param Success - Whether the request succeeded
```

Comment:
Can you please change this to "result" instead of "success" to make it align with the actual method parameter?